junit-jupiter-api/src/main/java/org/junit/jupiter/api/DynamicContainer.java

```diff
@@ -83,5 +102,4 @@ private DynamicContainer(String displayName, Stream<? extends DynamicNode> child
 	public Stream<? extends DynamicNode> getChildren() {
 		return children;
 	}
-
```

Comment:
IIRC, it's our policy (albeit perhaps unspoken) to include a blank line before the closing brace.